charts/argo-cd/values.yaml

```diff
@@ -115,6 +115,20 @@ repoServer:
     #   cpu: 10m
     #   memory: 64Mi
 
+# -- Notifications Controller
+notificationsController:
+  # -- Whether to enable Notifications Controller
+  enabled: true
+
+# -- ApplicationSet Controller
+applicationsetController:
+  # -- Whether to enable ApplicationSet Controller
+  enabled: true
```

Comment:
I think the default should actually be disabled

Reply:
Then the defaults are different from upstream and we have to explicitly set this to true when running our diff comparison script.